### ISSUE TITLE
Review accessibility on the website

### ISSUE THREAD

**Issue by jakubcech** [iotaledger/documentation-platform#97]
### Description
We should review what we are missing with regards to something like WCAG 2.1. 
https://www.w3.org/TR/WCAG21/

**Comment by digital-zen** [comment]
Agreed. I recommend we use http://pa11y.org/ for a robust accessibility audit.

**Comment by jakubcech** [comment]
That looks interesting, cheers :)
